### PR TITLE
CORCI-885 build: Removing SLES-12, JHLI rpm repos. (#2133)

### PR DESCRIPTION
**Pull request by JohnMalmberg**
Patch an issue with the Leap-15 dockerfile certificates.

Remove the remaining files for SLES-12 which we can no longer support.

Remove the JHLI rpm repositories as ipmctl is now available from
the normal repositories for a distro.

Signed-off-by: John E Malmberg <john.e.malmberg@intel.com>

### CODE REVIEW

**Comment by JohnMalmberg** [comment]
The test failure is DAOS-4085.
This commit is doing some cleanup of the build files to prevent future issues.

File: utils/rpms/Makefile [hunk 1]
```diff
@@ -6,26 +6,6 @@ PATCHES  = scons_local-$(VERSION).tar.$(SRC_EXT)
 #https://github.com/rpm-software-management/mock/issues/384
 MOCK_OPTIONS := --disablerepo=sclo*
 
-# override allows the caller to set a value and the we append to it
-#override PR_REPOS         +=
```
Comment (brianjmurrell) [inline utils/rpms/Makefile:10]: Please don't remove these two lines.  They are there to remind people where they can add other component PR repos to build/test with.

**Comment by JohnMalmberg** [comment]
Test failure is DAOS-4085.

File: utils/rpms/Makefile (hunk 1 above)
Comment (JohnMalmberg) [reply utils/rpms/Makefile]: fixed

**Comment by JohnMalmberg** [comment]
Now in the Wiki for merging in release/0.9

**Pull request merged**